scenarios/fuel_test/src/AllEventsHandlers.ts

```diff
@@ -228,20 +228,14 @@ AllEvents.Mint.handler(async ({ event }) => {
   expectType<AssertSchemaType<typeof event.params, typeof mintSchema>>(true);
 });
 
-// Should throw with "Duplicate event detected: Mint2 for contract AllEvents on chain 0"
-// AllEvents.Mint2.handler(async ({ event }) => {
-//   mintSchema.assert(event.params)!;
-//   expectType<AssertSchemaType<typeof event.params, typeof mintSchema>>(true);
+// const burnSchema = S.object({
+//   subId: S.string,
+//   amount: SExtra.bigint,
+// });
+// AllEvents.Burn.handler(async ({ event }) => {
+//   burnSchema.assert(event.params)!;
+//   expectType<AssertSchemaType<typeof event.params, typeof burnSchema>>(true);
```

Comment:
What's happening with this test?

Reply:
Oh, I wanted to show the case when a handler is missing. I'll add it back in a future PR